pylint/reporters/text.py

```diff
@@ -132,8 +132,13 @@ def colorize_ansi(
     """
     # TODO: 3.0: Remove deprecated typing and only accept MessageStyle as parameter
     if not isinstance(msg_style, MessageStyle):
+        WARNING_MESSAGE = (
+            "In pylint 3.0, the colorize_ansi function of Text reporters will only "
+        )
+        WARNING_MESSAGE += "accept a MessageStyle parameter"
```

Comment:
Please don't use additional assignments, you can use parenthesis and implicit string concatenation I've fixed/suggested that three time already :smile:

Reply:
Oops, sorry.